test/e2e/tools.go

```diff
@@ -4,4 +4,5 @@ package e2e
 
 import (
 	_ "github.com/jteeuwen/go-bindata/go-bindata"
+	_ "github.com/zalando-incubator/stackset-controller/cmd/e2e"
```

Comment:
I'm curious why this is in a separate file. Is it because these aren't imported anywhere but the `run_e2e.sh` script?

Reply:
Just by convention. It's a "hack" to have the package managed with Go modules but we only use it to generate a separate binary in the `Makefile`.